pkgs/development/tools/glpaper/default.nix

```
@@ -0,0 +1,35 @@
+{ stdenv, fetchhg
+, meson, ninja
+, pkg-config
+, wlroots, wayland, wayland-protocols
+, pixman, libxkbcommon
+,libX11
+, libGL, libglvnd
+}:
+
+stdenv.mkDerivation {
+  name = "glpaper";
+  version = "a95db77088dfb5636a87f3743fc9b5eca70c1ae2";
+
+  src = fetchhg {
+    url = "https://hg.sr.ht/~scoopta/glpaper";
+    rev = "a95db77088dfb5636a87f3743fc9b5eca70c1ae2";
+    sha256 = "04y12910wvhy4aqx2sa63dy9l6nbs7b77yqpdhc96x2b3mgzgjfs";
+  };
+
+  nativeBuildInputs = [ meson ninja pkg-config ];
+  buildInputs = [
+    wlroots wayland wayland-protocols wlroots
+    pixman libxkbcommon
+    libX11
+    libGL libglvnd
+  ];
+
+  meta = with stdenv.lib; {
+    description = "GLPaper is a wallpaper program for wlroots based wayland compositors such as sway that allows you to render glsl shaders as your wallpaper";
+    homepage    = "https://bitbucket.org/Scoopta/glpaper";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Plus;
```

https://bitbucket.org/Scoopta/glpaper/src/a95db77088dfb5636a87f3743fc9b5eca70c1ae2/src/paper.c#lines-6:7